tox.ini

```diff
@@ -64,10 +64,11 @@ commands =
          # pylint src --jobs=0
          pylint src/probnum/diffeq --disable="protected-access,abstract-class-instantiated,too-many-locals,too-few-public-methods,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring" --jobs=0
          pylint src/probnum/filtsmooth --disable="duplicate-code,no-self-use,too-many-locals,too-many-arguments,unused-argument,missing-module-docstring,missing-function-docstring" --jobs=0
+         pylint src/probnum/kernels --disable="too-few-public-methods,invalid-name" --jobs=0
          pylint src/probnum/linalg --disable="attribute-defined-outside-init,too-many-statements,too-many-instance-attributes,too-complex,protected-access,too-many-lines,no-self-use,too-many-locals,redefined-builtin,arguments-differ,abstract-method,too-many-arguments,too-many-branches,duplicate-code,unused-argument,fixme,missing-module-docstring" --jobs=0
-         pylint src/probnum/linops --jobs=0
+         pylint src/probnum/linops --disable="attribute-defined-outside-init,too-many-statements,too-many-instance-attributes,too-complex,protected-access,too-many-lines,no-self-use,too-many-locals,redefined-builtin,arguments-differ,abstract-method,too-many-arguments,too-many-branches,duplicate-code,unused-argument,fixme,missing-module-docstring" --jobs=0
```

Comment:
where does this explosion come from? It doesnt seem to me that there was a lot of change in linops (if any?).

Reply:
When `linops` was refactored out of `linalg` the exceptions were still there, but are not caught by `pylint`. In a future refactor these have to be removed one by one.